cloudbuild.yaml

```diff
@@ -7,20 +7,35 @@ steps:
 - name: 'gcr.io/cloud-builders/git'
   args:
   - 'checkout'
-  - 'tags/v0.1.2'
+  - 'tags/v0.2.0-m2'
   dir: 'qupath'
-- name: 'maven:3.6.1-amazoncorretto-8'
+- name: 'bash:5.0.7'
+  args:
+  - 'sed'
+  - '-i'
+  - '164a\apply plugin: "maven"'
+  - '/workspace/qupath/build.gradle'
+- name: 'gradle:5.4.1-jdk11'
   volumes:
   - name: 'vol1'
     path: '/root/.m2'
   args:
-  - 'mvn'
+  - 'gradle'
```

Comment:
Why maven to gradle?

Reply:
Because the new version of QuPath is based on Gradle.

Reply:
Ah ok thanks.